bundles/org.openhab.binding.modbus/pom.xml

```diff
@@ -14,13 +14,4 @@
 
   <name>openHAB Add-ons :: Bundles :: Modbus Binding</name>
 
-  <dependencies>
-    <dependency>
-      <groupId>org.openhab.addons.bundles</groupId>
```

Comment:
We have many modbus bindings (e.g. Sunspec, e3dc), not just the "generic" Modbus binding.

Should pom of others be updated as well?

Reply:
Good catch, I have updated all of them now.

Reply:
Follow up : I can see that here we have removed the dependency from pom.xml all together. In other modbus bindings we have it changed to point to the new group id in core.

Is this intentional and correct?

Reply:
I have removed the dependency altogether, because it is implicitly available for compilation, since it is in the core feature.